content/en/developers/write_agent_check.md

```diff
@@ -63,7 +63,7 @@ For more details about the interface provided by the base class, browse the [API
 To change the collection interval of your check, use `min_collection_interval` in the configuration file. The default value is `15` which means the `check` method from your class is invoked with the same interval as the rest of the integrations on the Agent.
 
 {{< tabs >}}
-{{% tab "Agent v6" %}}
+{{% tab "Agent v6 & v7" %}}
```

Comment:
We should add a note on this page about which Python version should be used to write the custom check for different Agent versions (Python 3 for Agent 7 / 6.14+, Python 2 for Agent 5).

Reply:
Pushed an update to write a check + https://github.com/DataDog/integrations-core/pull/5235